security/app-layer-policy.md

```diff
@@ -16,11 +16,15 @@ Enable {{site.prodname}} application layer network policy in Istio service mesh.
 
 #### Mitigate threats with {{site.prodname}} network policy
 
-Although Istio policy is ideal for operational goals, security inside and outside the cluster requires {{site.prodname}} network policy. {{site.prodname}} supports a special integration for Istio, called **application layer policy**. This policy lets you restrict ingress traffic inside and outside pods, and mitigate common threats to Istio-enabled apps.
+Although Istio policy is ideal for operational goals, security inside and outside the cluster requires network policy. {{site.prodname}} supports a special integration for Istio called, **application layer policy**. This policy lets you restrict ingress traffic inside and outside pods, and mitigate common threats to Istio-enabled apps. Using {{site.prodname}} network policy with Istio lets you adopt a zero trust network model for security, including traffic encryption, multiple enforcement points, and multiple identity criteria for authentication.
```

Comment:
I would skip some if not all of this. The opening is quite confusing ("Istio policy is ideal for operational goals"). I'm in two minds whether we want to talk about enforcement at the Istio layer as ALP, since you don't need to make any changes to network policy for them to be enforced by Istio. It's just a different enforcement point. If we get rid of ALP terminology in this doc, we probably need to search for the term in the rest of the docs and rename those to use the enforce within service mesh terminology.

I would add in the other value point from the other doc: "Support security goals
Using Calico network policy with Istio enables adoption of a zero trust network model for security, including traffic encryption, multiple enforcement points, and multiple identity criteria for authentication."

Reply:
Rewrite. WDYT?